cli/cmd/summary.go

```diff
@@ -5,22 +5,33 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"time"
+
+	"github.com/cloudquery/cloudquery/cli/internal/specs/v0"
+	"github.com/cloudquery/cloudquery/cli/internal/transformer"
+	"github.com/cloudquery/plugin-pb-go/managedplugin"
+	"github.com/cloudquery/plugin-pb-go/pb/plugin/v3"
+	"github.com/cloudquery/plugin-sdk/v4/caser"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+	"github.com/thoas/go-funk"
 )
 
 type syncSummary struct {
-	CliVersion          string `json:"cli_version"`
-	DestinationErrors   uint64 `json:"destination_errors"`
-	DestinationName     string `json:"destination_name"`
-	DestinationPath     string `json:"destination_path"`
-	DestinationVersion  string `json:"destination_version"`
-	DestinationWarnings uint64 `json:"destination_warnings"`
-	Resources           uint64 `json:"resources"`
-	SourceErrors        uint64 `json:"source_errors"`
-	SourceName          string `json:"source_name"`
-	SourcePath          string `json:"source_path"`
-	SourceVersion       string `json:"source_version"`
-	SourceWarnings      uint64 `json:"source_warnings"`
-	SyncID              string `json:"sync_id"`
+	CLIVersion          string    `json:"cli_version"`
+	DestinationErrors   uint64    `json:"destination_errors"`
+	DestinationName     string    `json:"destination_name"`
+	DestinationPath     string    `json:"destination_path"`
+	DestinationVersion  string    `json:"destination_version"`
+	DestinationWarnings uint64    `json:"destination_warnings"`
+	Resources           uint64    `json:"resources"`
+	SourceErrors        uint64    `json:"source_errors"`
+	SourceName          string    `json:"source_name"`
+	SourcePath          string    `json:"source_path"`
+	SourceVersion       string    `json:"source_version"`
+	SourceWarnings      uint64    `json:"source_warnings"`
+	SyncID              string    `json:"cq_sync_id"`
+	SyncTime            time.Time `json:"cq_sync_time"`
```

Comment:
```suggestion
	SyncID              string    `json:"sync_id"`
	SyncTime            time.Time `json:"sync_time"`
```

Unrelated to this PR and non blocking but maybe we should drop the `cq_` prefix as we don't use it anywhere in this struct, nor it's needed if the data goes into a dedicated file/table